Microsoft.Azure.Cosmos/src/Telemetry/ClientTelemetry.cs

```diff
@@ -320,10 +336,41 @@ private async Task EnrichAndSendAsync()
             }
             catch (Exception ex)
             {
-                DefaultTrace.TraceError("Request Charge Recording Failed by Telemetry. Request Charge Value : {0}  Exception : {1} ", requestChargeToRecord, ex.Message);
+                DefaultTrace.TraceError($"Request Charge Recording Failed by Telemetry. Request Charge Value : {requestChargeToRecord}  Exception : {ex} ");
```

Comment:
Does `requestChargeToRecord` implement `ToString()` method?

```suggestion
                DefaultTrace.TraceError("Request Charge Recording Failed by Telemetry. Request Charge Value : {0}  Exception : {1}", requestChargeToRecord, ex);
```

Reply:
its `long ` type value.